CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+# Contributing to the Synthetics Agent
+
+The Elastic Synthetics Agent is open source and we love to receive contributions from our community — you!
+
+There are many ways to contribute, from writing tutorials or blog posts,
+improving the documentation, submitting bug reports and feature requests or
+writing code.
+
+You can get in touch with us through [Discuss](https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics)—
+feedback and ideas are always welcome.
+
+## Code contributions
+
+If you have a bugfix or new feature that you would like to contribute,
+please find or open an issue about it first. Talk about what you would like to do.
+It may be that somebody is already working on it,
+or that there are particular issues that you should know about before implementing the change.
+
+### Submitting your changes
+
+Generally, we require that you test any code you are adding or modifying.
+Once your changes are ready to submit for review:
+
+1. Sign the Contributor License Agreement
+
+   Please make sure you have signed our [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
+   We are not asking you to assign copyright to us, but to give us the right to
+   distribute your code without restriction. We ask this of all contributors in
+   order to assure our users of the origin and continuing existence of the code.
+   You only need to sign the CLA once.
+
+2. Test your changes
+
+   Run the test suite to make sure that nothing is broken. If you're adding new
+   code or changing existing code, write some automated tests that exercise this
+   code. See [testing](#testing) for details.
+
+3. Rebase your changes
```

Comment:
It doesn't really matter whether they rebase vs merge anymore. I set the github repo to only allow the squash+merge button.

Reply:
But it's a good practise to rebase anyways right? We squash them anyways at the end.